test/commands/distribute/release/release-test.ts

```diff
@@ -846,14 +846,6 @@ describe("release command", () => {
       });
   }
 
-  function setupFailedCreateReleaseResponse(nockScope: Nock.Scope): Nock.Scope {
```

Comment:
Why is it removed?

Reply:
Unused

Reply:
Must have been left from previous implementation